### PR TITLE
Global displacement

### PR DESCRIPTION
**Pull request by aleesteele**
Basic global displacement chart, no styling

### CODE REVIEW

**Review by jnschrag — CHANGES_REQUESTED** [review]
- Since we're starting from scratch with these, can you go ahead and remove the jQuery dependency?
- Please update the `<title>` tag with the title of the chart
- The zoom in feature is nice to have, however, the X axis can start having decimals on them. Can you look at making sure the year numbers stay whole numbers?

**Comment by aleesteele** [comment]
- refactored for vanilla javascript 
- deleted jquery label & removed library
- kept x-axis whole numbers (years)
- rounded point values & removed extra x-axis label
- better refactoring (see most recent commit)

Q's: 
- should I add 'millions' to the point labels themselves? 
- should y-axis labels remain uppercase 'M''s or lowercase? (caroline answered!)

**Comment by aleesteele** [comment]
- Horizontal wide gridlines
- Capitalized y-axis labels / custom labels w/ mil
- Normal weight legend

Q's:
- Didn't change # of x-axis ticks, there might be too much

File: globals/ppd-highcharts-theme.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      font: '70px',
+      textTransform: 'uppercase',
+      fontWeight: 'bold'
+    }
+  },
+  tooltip: {
+    style: {
+      fontSize: "13px",
+    },
+    headerFormat: "<span style=\"font-size: 13px;text-align:center;margin-bottom: 5px;font-weight: bold;font-family: 'Source Sans Pro', arial, sans-serif;\">{point.key}</span><br/>"
```
Comment (jnschrag) [inline globals/ppd-highcharts-theme.js]: Let's remove the reference to "Source Sans Pro" here and make it Proxima Nova like the rest of the chart.

**Review by jnschrag — CHANGES_REQUESTED** [review]
This is really good, nicely done! Just a few small tweaks and then I'll accept it.

- Can you make sure that in `index.html` all of the external sources have `https:` at the beginning of their URLs?
- Make a copy of the iframeSizer and put it in PPD's global folder, then include it in the chart's `index.html`
- Add "People" as a value suffix in the tooltip

**Comment by aleesteele** [comment]
Fixed this one up too!

**Review by jnschrag — APPROVED** [review]

**Pull request merged**